### PR TITLE
amyyalex's card

### PR DESCRIPTION
**Pull request by amyyalex**
Added my card information to the index.html file

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -115,6 +115,33 @@ <h2>A project for first-time contributions</h2>
...
         <!-- ========= Paste YOUR CARD directly BELOW this line ========= -->
 
+         <!-- ________ amyyalex card START ________  -->
+         <div class="card">
+          <p class="name">amyyalex</p>
+          <p class="contact">
+            <i class="fab fa-twitter"></i>
+            <a href="https://twitter.com/AlexStephanie10" target="_blank">@AlexStephanie10</a>
+            <i class="fab fa-github"></i>
+            <a href="https://github.com/amyyalex" target="_blank">Github</a>
```
Comment (TenTraicion) [inline index.html:125]:
```suggestion
            <a href="https://github.com/amyyalex" target="_blank">@amyyalex</a>
```

**Review by TenTraicion — CHANGES_REQUESTED** [review]
@amyyalex, fix the handle of your social media.

File: index.html (hunk 1 above)
Comment (amyyalex) [reply index.html]: I just corrected it, thanks for pointing that out

**Review by TenTraicion — APPROVED** [review]

**Comment by TenTraicion** [comment]
@amyyalex, Congratulation on your PR Merge, and Thanks for your Contribution. You can check your card in our [Live Website](https://syknapse.github.io/Contribute-To-This-Project/)! It may take 2-3 minutes to make your card visible in our website. 

Welcome to Open Source Community. You can follow the [Next Steps](https://github.com/Syknapse/Contribute-To-This-Project#next-steps) now. You are also welcome to join our [Discord Server: https://discord.gg/tWkvS4ueVF](https://discord.gg/tWkvS4ueVF) and be a part of our community.

**Pull request merged**